TAF/testCaseModules/keywords/support-notifications/subscriptionAPI.robot

```diff
@@ -51,18 +51,16 @@ Create Subscription ${entity}
     ${headers}=  Create Dictionary  Content-Type=application/json  Authorization=Bearer ${jwt_token}
     ${resp}=  POST On Session  Support Notifications  ${subscriptionUri}  json=${entity}   headers=${headers}
     ...       expected_status=any
-    Run Keyword If  "${api_version}" == "v1"  Run Keywords
-    ...             Run keyword if  ${resp.status_code}!=200  log to console  ${resp.content}
-    ...             AND  Set Test Variable  ${response}  ${resp.status_code}
-    ...    ELSE IF  "${api_version}" == "v2"  Run Keywords  Set Response to Test Variables  ${resp}
-    ...             AND  Run keyword if  ${response} != 207  log to console  ${content}
+    Set Response to Test Variables  ${resp}
+    Run keyword if  ${response} != 207  log to console  ${content}
 
 Delete Subscription By Name ${subscriptionName}
     Create Session  Support Notification  url=${supportNotificationsUrl}  disable_warnings=true
     ${headers}=  Create Dictionary  Authorization=Bearer ${jwt_token}
     ${resp}=  DELETE On Session  Support Notification  ${subscriptionUri}/name/${subscriptionName}  headers=${headers}
     ...       expected_status=any
     run keyword if  ${resp.status_code}!=204  log to console  ${resp.content}
+    Set Suite Variable  ${response}  ${resp.status_code}
```

Comment:
```suggestion
    Set Response to Test Variables ${resp}
```
This keyword returns `${response}`, `${headers}`, `${response_time}` and `${content}`

Reply:
Use `set suite variable` temporarily after the [edgex-go issue #3308](https://github.com/edgexfoundry/edgex-go/issues/3308) is fixed.

Reply:
Fix. Opened [ edgex-go issue #3308 ](https://github.com/edgexfoundry/edgex-go/issues/3308) for delete API error.